source/_integrations/duckdns.markdown

```diff
@@ -29,7 +29,7 @@ duckdns:
 
 {% configuration duckdns %}
   domain:
-    description: Your duckdns subdomain (without the `.duckdns.org` suffix).
+    description: Your duckdns subdomain (with the `.duckdns.org` suffix).
```

Comment:
The link to reddit is about the add-on, this documentation is about the integration.
Are you sure you aren't mixing things up?

Could you point me to upstream documentation that shows this being needed for the integration as well?